Index.html

```diff
@@ -248,9 +248,9 @@ <h3 class="text-content-4">
             like to chat please get in touch.
           </h3>
           <form class="feedback-form">
-            <input class="inp" placeholder="Full name" />
-            <input class="inp" placeholder="Email address" />
-            <textarea class="text-area">Write me something...</textarea>
+            <input type="text" maxlength="30" class="inp" placeholder="Full name" />
+            <input type="email" class="inp" placeholder="Email address" />
+            <textarea maxlength="500" class="text-area" placeholder="Write me something..."></textarea>
```

Comment:
- [ ] Nice but you are missing the `required` parameter in all fields as asked in the requirement, please think to add it
- [ ] Your form can not submit, you don't have form spree implemented, please read well the requirement @MohammadYaser 
- [ ] This green space along your app is not part of the design, your content needs to fit the page as here in the original [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=UFHIK0VKK8PfjXdk-0)

YOUR DESIGN
<img width="454" alt="1" src="https://github.com/MohammadYaser/Portfolio/assets/77843050/b11a630d-7ce6-49ea-a53a-1e932875417c">

FIGMA
<img width="513" alt="1" src="https://github.com/MohammadYaser/Portfolio/assets/77843050/dab74a1a-02e6-41c2-985e-15ea93ac4de8">